great-idea/css/index.css

```diff
@@ -63,4 +63,136 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Copy and paste your work from yesterday here and start to refactor into flexbox */
\ No newline at end of file
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+
+.main{
+	width: 1000px;
+	margin: 0 auto;
+}
+.header{
+	display: flex;
+	justify-content: space-around;
+}
+
+.header nav{
+	display: flex;
+	justify-content: space-around;
+	flex-basis: 80%;
+
+}
+ nav a {
+	 padding: 20px;
+	 color: grey;
+ }
+
+
+ .get{
+	 border: 1px solid black;
+	 width: 120px;
+	 text-align: center;
+	 padding: 10px;
+	 margin-left: 50px;
+	 
+	 
+	
+ }
+
+ #company-name{
+	 font-size: 65px;
+	 align-items: center;
+	 margin-left: 20px;
+
+ }
+
+ 
+ .first-section{
+	 display: flex;
+	 justify-content: space-around;
+	 align-items: center;
+	 margin-top: 40px;
+
+ }
+
+ .image{
+	 margin-left: 200px;
+	 margin-right: 30px;
+ }
+ .second-section{
+	 display: flex;
+	 margin: 0 auto;
+	 margin-bottom: 30px;
+	 margin-top: 30px;
+	 margin-right: 40px;
+	 justify-content: space-between;
+ }
+
+ .third-section{
+	 display: flex;
+	 margin: 0 auto;
+	 justify-content: space-between;
+	 margin: 30px 40px 20px 0px;
+ }
+.copy{
+	text-align: center;
+	margin-top: 40px;
+}
+.text{
+	line-height: 30px;
+}
+
+.outlay, .outlay2, .outlay3{
+	display: flex;
+	margin: 0 auto;
+	justify-content: space-between;
+}
+.marketing, .design, .ux, .web, .ios, .android{
+	border: 1px solid black;
+	margin: 15px;
+	padding:10px;
+	background-color: lightgray;
+	
+}
+.learn{
+	border: 1px solid black;
+	width: 150px;
+	text-align: center;
+	border-radius: 5px;
+	padding: 2px;
+	margin-top: 20px;
+	margin-bottom: 10px;
+	background-color: white;
+	height: 30px;
+}
+
+.outlay{
+	margin-top: 18px;
+}
+
+.lower-section{
+	display: flex;
+	margin: 0 auto;
+	justify-content: space-between;
+}
+
+services-info-img{
+	margin: 10px 15px 15px 20px;
+}
```

Comment:
This image appears off balance, try giving it a width of auto so that it takes the entire space of the container.